swift-sdk/Internal/IterableAPIInternal.swift

```diff
@@ -606,7 +606,7 @@ final class IterableAPIInternal : NSObject, PushTrackerProtocol {
                 toLog += ", \(reason)"
             }
             if let data = data {
-                toLog += ", got response \(data)"
+                toLog += ", got response \(String(data: data, encoding: .utf8))"
```

Comment:
```suggestion
                toLog = ", got response \(String(data: data1, encoding: .utf8) ?? "nil")"
```

Reply:
This will get rid of warning since String(data:encoding:) returns optional String.